README.md

```diff
@@ -1,37 +1,173 @@
 
-Minimal static [SvelteKit](https://kit.svelte.dev/) set-up made deployable to [GitHub Pages](https://metonym.github.io/sveltekit-gh-pages/). Based on [sveltekit-gh-pages](https://metonym.github.io/sveltekit-gh-pages/)
+![Spyder Logo](static/assets/media/spyder-logomark.png)
 
-- PostCSS framework provided by [TailwindCSS](https://tailwindcss.com/)
-- Icons from the [Svelte Icons Pack](https://leshak.github.io/svelte-icons-pack/)
-- Markdown files processed with [mdsvex](https://mdsvex.pngwn.io/)
+The official website for [Spyder IDE](https://www.spyder-ide.org) - a powerful scientific environment written in Python, for Python, and designed by and for scientists, engineers and data analysts.
 
-## Deployment
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![Built with SvelteKit](https://img.shields.io/badge/Built%20with-SvelteKit-orange.svg)](https://kit.svelte.dev/)
+[![Deployed on GitHub Pages](https://img.shields.io/badge/Deployed%20on-GitHub%20Pages-blue.svg)](https://pages.github.com/)
 
-### 1) Install dependencies with NPM:
+🌐 **Live Site**: [www.spyder-ide.org](https://www.spyder-ide.org)
 
-```bash
-npm install
-```
+## About
 
-### 2) Test the website (optional)
+This repository contains the source code for the official Spyder IDE website. The site serves as:
 
-```bash
-npm run dev
-```
+- **Information hub** for Spyder features and capabilities
+- **Download center** for installers and releases
+- **Blog platform** for project updates and community news
+- **Documentation gateway** linking to guides and resources
+- **Community showcase** highlighting contributors and achievements
 
-And visit your local copy at [https://localhost:5173](https://localhost:5173)
+## Tech Stack
 
-### 3) Build the website
+This website is built with modern web technologies:
 
-```bash
-npm run build
+- **Framework**: [SvelteKit](https://kit.svelte.dev/) - Full-stack web framework
+- **Styling**: [TailwindCSS](https://tailwindcss.com/) - Utility-first CSS framework
+- **Content**: [mdsvex](https://mdsvex.pngwn.io/) - Markdown processor for Svelte
+- **Icons**: [Svelte Icons Pack](https://leshak.github.io/svelte-icons-pack/) - Icon library
+- **Deployment**: [GitHub Pages](https://pages.github.com/) - Static site hosting
+- **CI/CD**: GitHub Actions - Automated deployment
+
+## Features
+
+- ✨ **Modern Design** - Clean, responsive interface
+- 📱 **Mobile Optimized** - Works perfectly on all devices
+- 🚀 **Fast Performance** - Static site generation for speed
+- 📝 **Blog System** - Dynamic blog with author management
+- 🎨 **Component Library** - Reusable UI components
+- 🌍 **Internationalization** - Multi-language support ready
+- ♿ **Accessibility** - WCAG compliant design
+- 🔍 **SEO Optimized** - Meta tags and structured data
+
+## Quick Start
+
+### Prerequisites
+
+- [Node.js](https://nodejs.org/) (version 18 or higher)
+- [npm](https://www.npmjs.com/) (comes with Node.js)
```

Comment:
This should be in the contributing guide (as this should be a subset of content there)